### ISSUE TITLE
Many wordlists same time

### ISSUE THREAD

**Issue by hugonh** [ffuf/ffuf#512]
Good afternoon!

I would like to know how can i add many wordlists at same time, and for example, a new job is found in recursive mode, i would like that new job could be tested on next wordlist too, for more jobs(fuzzing entries)

How can i do that, please?

Example: 
ffuf  -w *.txt -u **/FUZZ -rate 20 -maxtime-job 5000 -recursion -recursion-depth 3 -p 0.1-5.0

 * = wordlist
 * * = website url

**Comment by joohoi** [comment]
I don't think this is possible with ffuf currently. Your best bet would probably be saving the output to a JSON file or such, parse that and create a new job with the secondary wordlist based on the results.
